src/Microsoft.AspNetCore.Mvc.WebApiCompatShim/ContentNegotiator/MediaTypeConstants.cs

```diff
@@ -101,7 +101,7 @@ public static MediaTypeHeaderValue ApplicationFormUrlEncodedMediaType
         /// </value>
         /// <remarks>
         /// Not yet a standard. In particular this media type is not currently listed at
-        /// http://www.iana.org/assignments/media-types/application.
+        /// https://www.iana.org/assignments/media-types/application.
```

Comment:
Good news is http://www.w3.org redirects to the SSL endpoint. The standards balance out 😈